### PR TITLE
new recipe colored

### PR DESCRIPTION
**Pull request by ozcan**
Homepage:  https://gitlab.com/dslackw/colored

### CODE REVIEW

**Comment by ozcan** [comment]
Hi, Is Windows test necessary? If not can someone help me to merge this, i have another recipe pending because it has this module as a dependency

File: recipes/colored/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  imports:
+    - colored
+
+about:
+  home: https://gitlab.com/dslackw/colored
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: Simple library for color and formatting to terminal
+  doc_url:
```
Comment (xhochy) [inline recipes/colored/meta.yaml]: If these attributes are empty, please omit them fully.

File: recipes/colored/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+test:
+  imports:
+    - colored
+
+about:
+  home: https://gitlab.com/dslackw/colored
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (xhochy) [inline recipes/colored/meta.yaml]: Please ensure that a license file is packaged.

File: recipes/colored/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8296ea990e3f6b7822f44eec21408b126dfb9c1c031306b859e3f7d46cc27075
+
+build:
+  number: 0
```
Comment (xhochy) [inline recipes/colored/meta.yaml]: Please also add `noarch: python` as the builds of this package don't depend on the Python version.

**Comment by meren** [comment]
Thank you very much for your help, @xhochy! :)

**Comment by xhochy** [comment]
@conda-forge/staged-recipes This is ready for review/merge.

**Pull request merged**